src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dj/DefaultJudgmentOrderFormGenerator.java

```diff
@@ -239,17 +239,28 @@ private String checkDefendantRequested(final CaseData caseData) {
         }
     }
 
-    private String fillTypeBundleInfo(DisposalHearingBundleType type) {
-        switch (type) {
-            case DOCUMENTS:
-                return "an indexed bundle of documents, with each page clearly numbered.";
-            case ELECTRONIC:
-                return "an electronic bundle of digital documents.";
-            case SUMMARY:
-                return "a case summary containing no more than 500 words.";
-            default:
-                return null;
+    private static String fillTypeBundleInfo(CaseData caseData) {
+        DisposalHearingBundleDJ disposalHearingBundle = caseData.getDisposalHearingBundleDJ();
+
+        if (disposalHearingBundle != null) {
+            List<DisposalHearingBundleType> types = disposalHearingBundle.getType();
+            StringBuilder stringBuilder = new StringBuilder();
+
```

Comment:
I don't know if it would be readable enough, but you can also do something like 
stringBuilder.append(Stream.of(DisposalHearingBundle.values()).map(DisposalHearingBundleType::getLabel).collect(Collectors.joining(" / "))) and that will work if we add/remove elements from DisposalHearingBundleType